src/kt_class.cpp

```diff
@@ -99,6 +99,15 @@ void KtClass::fetch_methods(jni::Env& env) {
     }
 }
 
+void KtClass::fetch_rpc_methods() {
+    HashMap<StringName, KtFunction*>::Iterator current = methods.begin();
+    while (current) {
```

Comment:
With 4.0 you can now use classic for each, like here : https://github.com/utopia-rise/godot-kotlin-jvm/blob/d108e232d4b0599ba53aca3d54ed382e653fdcb2/src/gd_kotlin.cpp#L456